.gitignore

```diff
@@ -9,5 +9,5 @@ __pycache__/
 *.log
 logs/
 .env
-.venv/
```

Comment:
Why is this change required ? Can you move it to a different PR if you have a valid reasoning for it ?